Wrapping/Python/Python.i

```diff
@@ -577,9 +585,17 @@
 
 
         def __setitem__( self, idx, value ):
-            """Sets the pixel value at index idx to value.
+            """Sets pixel value(s) values.
```

Comment:
what does 'value(s) values' mean?  is value supposed to be there twice?

Reply:
I renamed the argument to rvalue.